Admin/Infrastructure/Proxmox/1-InstallingProxmox+Infrastructure Specific Details.md

```diff
@@ -1,8 +1,8 @@
 #Installing Promox + Infrastructure Specific Details
 
-Author:
+*Author(s):*

```

Comment:
Similar Suggestion to the one in the `3-Removing Nodes` and `2-ChangingHostNames` document.

```suggestion
**Author(s)**: 
* Chisom
* Luke
---
```
* This way "Authors: is bolded 
* The Author's name is indented with a bullet point (Could omit the bullet point)
* The `---` inserts a line